src/components/Sum.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Sum.scss';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line arrow-parens
```

Comment:
why you need this rule?

Reply:
because I want to use single function argument :disappointed: